src/components/Teams/TeamMembersPopup.jsx

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable react/prop-types */
+import React, { useState, useEffect } from 'react';
+import {
+  Button, Modal, ModalHeader, ModalBody, ModalFooter, Container, Alert,
+} from 'reactstrap';
+import MembersAutoComplete from './MembersAutoComplete';
+
+const TeamMembersPopup = React.memo((props) => {
+  const closePopup = () => { props.onClose(); };
+  const [selectedUser, onSelectUser] = useState(undefined);
+  const [isValidUser, onValidation] = useState(true);
+  const onAddUser = () => {
+    if (selectedUser && !props.members.teamMembers.some((x) => x._id === selectedUser._id)) {
+      props.onAddUser(selectedUser);
+    } else {
+      onValidation(false);
+    }
+  };
+  const selectUser = (user) => {
+    onSelectUser(user);
+    onValidation(true);
+  };
+
+  useEffect(() => {
+    onValidation(true);
+  }, [props.open]);
+
+  return (
+    <Container fluid>
+      <Modal isOpen={props.open} toggle={closePopup}>
+        <ModalHeader toggle={closePopup}>
+          {`Members of ${props.selectedTeamName}`}
+        </ModalHeader>
+        <ModalBody style={{ textAlign: 'center' }}>
+          <div className="input-group-prepend" style={{ marginBottom: '10px' }}>
+            <MembersAutoComplete
+              userProfileData={props.usersdata}
+              onAddUser={selectUser}
+            />
+            <Button
+              color="primary"
+              onClick={onAddUser}
+            >
+              Add
+            </Button>
+          </div>
+          {(isValidUser === false) ? (
+            <Alert color="danger">
+              Please choose a valid user.
+            </Alert>
+          ) : <></>}
+          <div>
+            <table className="table table-bordered table-responsive-sm">
+              <thead>
+
+                <tr>
+                  <th>#</th>
+                  <th>User Name</th>
+                  <th>
+                    {' '}
+                  </th>
+                </tr>
+              </thead>
+              <tbody>
+                {props.members.teamMembers.length > 0
+                  ? props.members.teamMembers.map((user, index) => (
```

Comment:
Each child in a list should have a unique "key" prop.

Reply:
Corrected